homeassistant/components/linkplay/utils.py

```diff
@@ -25,10 +25,12 @@
 MODELS_ARYLIC_A50: Final[str] = "A50"
 MODELS_ARYLIC_A50S: Final[str] = "A50+"
 MODELS_ARYLIC_UP2STREAM_AMP: Final[str] = "Up2Stream Amp 2.0"
+MODELS_ARYLIC_UP2STREAM_AMP_2P1: Final[str] = "Up2Stream Amp 2.1"
```

Comment:
Can you open a preliminary pull request to add these models? Pull requests are as small as possible and should only address one change at a time.

Reply:
Thanks, I've just submitted a separate PR with these.